README.md

```diff
@@ -29,13 +29,14 @@ $ conda install -c conda-forge kneed
 
 **pip**
 ```bash
-$ pip install kneed
+$ pip install kneed # To install only knee-detection algorithm
+$ pip install kneed[plot] # To also install plotting functions for quick visualizations
 ```
 
 **Clone from GitHub**
 ```bash
 $ git clone https://github.com/arvkevi/kneed.git
-$ python setup.py install
```

Comment:
I never caught this, should probably add a line here to change into the kneed directory.  
`cd kneed`

Reply:
Yes, I did not caught it but it is quite obvious this is a typo. 

For the CONTIBUTING.md change, I fixed item list 3 as you requested but probably the item 2 will need some fixes too in future PR (the venv activation command is missing)